tests/test_commands.py

```diff
@@ -155,100 +155,70 @@ def test_generate_transform_commands_all_input_fields(etl_version_2, run_id):
     }
 
 
-def test_generate_load_commands_daily():
-    transform_output_files = [
-        "testsource/testsource-2022-01-02-daily-transformed-records-to-index_01.json",
-        "testsource/testsource-2022-01-02-daily-transformed-records-to-index_02.json",
-        "testsource/testsource-2022-01-02-daily-transformed-records-to-delete.txt",
-    ]
+def test_generate_load_commands_daily(etl_version_2, run_id):
     assert commands.generate_load_commands(
-        transform_output_files, "daily", "testsource", "test-timdex-bucket"
+        "testsource",
+        "2022-01-02",
+        "daily",
+        "run-abc-123",
+        "test-timdex-bucket",
     ) == {
-        "files-to-index": [
-            {
-                "load-command": [
-                    "bulk-index",
-                    "--source",
-                    "testsource",
-                    "s3://test-timdex-bucket/testsource/"
-                    "testsource-2022-01-02-daily-transformed-records-to-index_01.json",
-                ]
-            },
-            {
-                "load-command": [
-                    "bulk-index",
-                    "--source",
-                    "testsource",
-                    "s3://test-timdex-bucket/testsource/"
-                    "testsource-2022-01-02-daily-transformed-records-to-index_02.json",
-                ]
-            },
-        ],
-        "files-to-delete": [
-            {
-                "load-command": [
-                    "bulk-delete",
-                    "--source",
-                    "testsource",
-                    "s3://test-timdex-bucket/testsource/"
-                    "testsource-2022-01-02-daily-transformed-records-to-delete.txt",
-                ]
-            }
-        ],
+        "bulk-update-command": [
+            "bulk-update",
+            "--run-date",
+            "2022-01-02",
+            "--run-id",
+            "run-abc-123",
+            "--source",
+            "testsource",
+            "s3://test-timdex-bucket/dataset",
+        ]
```

Comment:
This test very explicitly demonstrates the new output.  Instead of nodes like `files-to-index` and `files-to-delete` that have standalone commands for TIM, we instead point TIM at a subset of the dataset which include _both_ records to index and delete.  TIM is then responsible for filtering that subset of records based on the `action` column and performing those operations.